### PR TITLE
Bottle Framework - test app

### PR DESCRIPTION
**Pull request by nappex**
- [x] add static files
- [x] ~~add dynamic routing ?? (I did not notice any solution in doc - quick search)~~ Dynamic routes are known also as wildcards - dynamically changing routes - match more than one URL
- [x] ~~add cycle and structured links~~

### CODE REVIEW

File: requirements-dev.txt [hunk 1]
```diff
@@ -3,3 +3,4 @@ flask
 pyflakes
 Django
 wsgi-static-middleware
+bottle == 0.12.19
```
Comment (encukou) [inline requirements-dev.txt:6]: Proč tuhle konkrétní verzi?

**Comment by encukou** [comment]
Looks like dynamic routing works similarly to Flask: https://bottlepy.org/docs/dev/routing.html

File: requirements-dev.txt (hunk 1 above)
Comment (nappex) [reply requirements-dev.txt:6]: hmm asi jsem to jen tupě zkopíroval :)

**Comment by nappex** [comment]
I supposed problem's occured because it depends on location where app has activated.
Attribute root is changing by that.

**Comment by nappex** [comment]
Route to image can be different from path in func static_files. But freezeyt freezing to path which read in URL.

**Comment by nappex** [comment]
❓ Do we want to add tox to requirements-dev.txt ? ❓

**Comment by encukou** [comment]
This looks good, but is still marked as work in progress. Do you want to make any other changes?

**Comment by nappex** [comment]
> This looks good, but is still marked as work in progress. Do you want to make any other changes?

No, I have not noticed

**Review by encukou — APPROVED** [review]

**Comment by encukou** [comment]
Great! Thank you!

**Pull request merged**